C++/C.sublime-syntax

```diff
@@ -111,6 +111,14 @@ contexts:
       scope: constant.other.placeholder.c
 
   keywords:
+    - match: \bbreak\b
+      scope: keyword.control.flow.break.c
+    - match: \bcontinue\b
+      scope: keyword.control.flow.continue.c
+    - match: \bgoto\b
+      scope: keyword.control.flow.goto.c
+    - match: \breturn\b
+      scope: keyword.control.flow.return.c
```

Comment:
Don't forget to mirror your changes in Objective-C and Objective-C++, too.

Reply:
It seems like the ObjC/C++ syntaxes already have these because of `- include: scope:source.c#keywords` so I updated the syntax tests (but excluded the language suffix from the check)